### PR TITLE
Updates + DIAlignR

### PR DESCRIPTION
**Pull request by Leon-Bichmann**
Basic updates and first draft of DIAlignR step

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [x] If necessary, also make a PR on the [nf-core/diaproteomics branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/diaproteomics)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`).
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [x] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/diaproteomics/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by ggabernet** [comment]
Hi Leon, it seems like there are some nf-core linting errors in environment.yml:

```
ERROR: Test Failures:
  http://nf-co.re/errors#8 : Conda dependency had an unknown version: conda-forge::python=3.6
  http://nf-co.re/errors#8 : Conda dependency did not have pinned version number: r-base
  http://nf-co.re/errors#8 : Conda dependency did not have pinned version number: networkx
  http://nf-co.re/errors#8 : Could not find Conda dependency using the Anaconda API: bioconda:easypqp=0.1.7
  http://nf-co.re/errors#8 : Conda dependency did not have pinned version number: bioconductor-dialignr

File: conf/test.config [hunk 1]
```diff
@@ -14,6 +14,15 @@ params {
   max_cpus = 2
   max_memory = 6.GB
   max_time = 48.h
+  pyprophet_peakgroup_fdr = 1
+  pyprophet_peptide_fdr = 1
+  pyprophet_protein_fdr = 1
+  DIAlignR_globalAlignFDR = 1
+  DIAlignR_AnalyteFDR = 1
+  DIAlignR_UnAlignFDR = 1
+  DIAlignR_AlignFDR = 1
```
Comment (ggabernet) [inline conf/test.config]:
```suggestion
  DIAlignR_global_align_FDR = 1
  DIAlignR_analyte_FDR = 1
  DIAlignR_unalign_FDR = 1
  DIAlignR_align_FDR = 1
```

Comment (ggabernet) [reply conf/test.config]: For readability, snake case is recommended when possible

File: main.nf [hunk 2]
```diff
@@ -45,11 +45,10 @@ def helpMessage() {
...
       --pyprophet_protein_fdr           Threshold for global Protein FDR
-      --realigment_target_fdr           Target FDR for realigned chromatograms
-      --realignment_max_fdr             Maximal FDR of realigned chromatograms
-      --realignment_score               Minimum scores of realigned chromatograms
-      --realignment_method              Method for realignment ('linear', 'lowess', 'lowess_cython')
-      --realignment_rt_difference       Maximal difference in RT across realigned chromatograms
+      --DIAlignR_globalAlignFDR         DIAlignR global Aligment FDR threshold
+      --DIAlignR_AnalyteFDR             DIAlignR Analyte FDR threshold
+      --DIAlignR_UnAlignFDR             DIAlignR UnAligment FDR threshold
+      --DIAlignR_AlignFDR               DIAlignR Aligment FDR threshold
```
Comment (ggabernet) [inline main.nf]:
```suggestion
      --DIAlignR_global_align_FDR         DIAlignR global Aligment FDR threshold
      --DIAlignR_analyte_FDR             DIAlignR Analyte FDR threshold
      --DIAlignR_unalign_FDR             DIAlignR UnAligment FDR threshold
      --DIAlignR_align_FDR               DIAlignR Aligment FDR threshold
```

File: nextflow.config [hunk 3]
```diff
@@ -25,12 +25,18 @@ params {
...
   pyprophet_fdr_ms_level = 'ms2'
   pyprophet_global_fdr_level = ''
   pyprophet_peakgroup_fdr = 0.01
   pyprophet_peptide_fdr = 0.01
   pyprophet_protein_fdr = 0.01
 
+  DIAlignR_globalAlignFDR = 0.01
+  DIAlignR_AnalyteFDR = 0.01
+  DIAlignR_UnAlignFDR = 0.01
+  DIAlignR_AlignFDR = 0.05
```
Comment (ggabernet) [inline nextflow.config]:
```suggestion
  DIAlignR_global_align_FDR = 0.01
  DIAlignR_analyte_FDR = 0.01
  DIAlignR_unalign_FDR = 0.01
  DIAlignR_align_FDR = 0.05
```

**Review by ggabernet — COMMENTED** [review]
The parameter names could be maybe improved a bit, but otherwise it looks fine to me once the tests pass

**Comment by Leon-Bichmann** [comment]
Yes makes sense, I renamed the params.

**Review by ggabernet — APPROVED** [review]
Looks good to me now!

**Pull request merged**